test/codeu/chat/util/TokenizerTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+* This class tests Tokenizer.java
+* author: Isaac Martins
+* version: May 19, 2017
+*/
+package codeu.chat.util;
+
+import java.io.IOException;
+import static org.junit.Assert.*;
+import org.junit.Test;
```

Comment:
You need to import Tokenizer here so that you can use it. They are not in the same place.